src/xrpld/rpc/detail/TransactionSign.cpp

```diff
@@ -222,6 +223,21 @@ checkPayment(
             rpcINVALID_PARAMS,
             "Cannot specify both 'tx_json.Paths' and 'build_path'");
 
+    std::optional<uint256> domain = std::nullopt;
```

Comment:
It will be std::nullopt by default

Reply:
a1171876e26dc5df9a4cfb5f12b05a685ae569ab